db/migrations/2_habits.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS habit_type;
+DROP TABLE IF EXISTS comp_dates;
+
+
+CREATE TABLE habit_type (
+    habit_id varchar REFERENCES users(habit_id),
+    userId varchar(25) REFERENCES users(id),
```

Comment:
if you are trying to use the foreign keys, use it consistently and then it should be referenced in the user table as well also syntax is incorrect you need to state that this is a foreign key that references certain table attribute. If you do not want to use foreign keys just don't use it, here you are trying to apply them sporadically.